tekton/resources.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: tekton-pipelines-v0-5-2
```

Comment:
i think naming this after a particular release will make it harder or more confusing to reuse this, can we give this a generic name instead?

Reply:
otherwise our instructions will need to tell ppl to create a new resource with the right name which imo is more complication than it's worth?